### PR TITLE
Render all products on the products page

### PR DESCRIPTION
**Pull request by nries1**
Please do not be alarmed when you see this! I did not add any styling yet.

Acceptance Criteria:
1) Pull the products-page branch
2) Start the app
3) Navigate to localhost:3000
4) Click on Shop
5) You should see all of our products rendered
6) Refresh the page
7) You should still see all of our products rendered

### CODE REVIEW

File: app/components/ProductPage.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import React from "react";
-import { Link } from "react-router-dom";
-import Button from "react-bootstrap";
-import product from "./Product.js";
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Button from 'react-bootstrap';
```
Comment (raymondng1) [inline app/components/ProductPage.js]: not really an issue but can always fix it later, import a { Button } or from 'react-bootstrap/Button'

**Review by raymondng1 — DISMISSED** [review]
Nice ! Looks like the redux store and dispatch worked great!

File: app/components/Products.js [hunk 2]
```diff
@@ -1,36 +1,53 @@
...
-            <option value="option 2">Option 2</option>
-          </select>
-          <select id="sort-by">
-            <option value="">Sort By</option>
-            <option value="option 1">Option 1</option>
-            <option value="option 2">Option 2</option>
-          </select>
+class Products extends React.Component {
+  componentDidMount() {
+    console.log(this.props);
```
Comment (raymondng1) [inline app/components/Products.js]: not sure if we need this console.log anymore

Comment (nries1) [reply app/components/Products.js]: you're right!

**Review by raymondng1 — APPROVED** [review]
Products page looks great!

**Pull request merged**